core/domain/classifier_domain.py

```diff
@@ -221,7 +221,7 @@ def training_data(self):
                     'answer_group_index': 2,
                     'answers': ['a2', 'a3']
                 }
-            ]
+            ].
```

Comment:
Hmm, had not thought of this case when checking punctuations. Please add this back into the punctuations whitelist.